### DISCUSSION TITLE
關於基建換班識別問題謝謝

### DISCUSSION THREAD

**Discussion by buzhidao33** [MaaAssistantArknights/MaaAssistantArknights#5432] [Q&A]
試了所有方法，基建換班的識別率還是很低，而且識別不到幹員後，它會一直在最右邊抽搐，一段時間後轉到最左邊抽搐，然後就退出來，如果能設置成再識別個2、3次就好很多，而且經常已經選對了，它不直接確定，而是取消所有選擇再選，再選的時候必漏選1個幹員，然後就出現上面的抽搐現象。。。現在每次基建我都要看著或過後檢查，幾乎沒有一次100%成功的。謝謝

**Comment by ABA2396** [comment]
请在[这里](https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/new?assignees=&labels=bug&projects=&template=cn-bug-report.yaml)发问题反馈并带上日志
